Rakefile

```diff
@@ -36,4 +36,11 @@ task lint: %i[rubocop environment] do
   sh "yarn run lint"
 end
 
-task default: [:lint, :spec, "app:jasmine:ci"]
+desc "Jasmine"
+task :jasmine do
+  sh "which chromedriver"
+  sh "chromedriver --version"
```

Comment:
Do we have a pain point that Ruby installs a chromedriver which isn't easy for node to find?

Reply:
yes, I think that was the case